exercises/practice/accumulate/src/lib.rs

```diff
@@ -1,4 +1,7 @@
 /// What should the type of _function be?
-pub fn map(input: Vec<i32>, _function: ???) -> Vec<i32> {
+pub fn map<F, T, U>(input: Vec<i32>, _function: F) -> Vec<i32>
+where
+    F: FnMut(T) -> U,
+{
```

Comment:
Strongly negative on this change. If rustfmt is complaining about it, we can add a meta-skip check for rustfmt for this exercise; see `.meta/ALLOWED_TO_NOT_COMPILE` for the compile checks for an example of how that's handled.

Reply:
Got it! I naively copied this but see that it should be invalid Rust code. I reverted this change and ensured `format_exercises` skips formatting on exercises with `.meta/ALLOWED_TO_NOT_COMPILE`